dojo/unittests/test_jfrogxray_parser.py

```diff
@@ -12,9 +16,22 @@ def test_parse_file_with_one_vuln(self):
         item = parser.items[0]
         self.assertEquals('debian:stretch:libx11', item.component_name)
         self.assertEquals('2:1.6.4-3', item.component_version)
+        self.assertEquals('CVE-2018-14600', item.cve)
+        self.assertEquals(787, item.cwe)
 
     def test_parse_file_with_many_vulns(self):
         testfile = open("dojo/unittests/scans/jfrogxray/many_vulns.json")
         parser = XrayJSONParser(testfile, Test())
         testfile.close()
         self.assertEqual(3, len(parser.items))
+
+    def test_parse_file_with_many_vulns2(self):
+        testfile = open("dojo/unittests/scans/jfrogxray/many_vulns2.json")
+        parser = XrayJSONParser(testfile, Test())
+        testfile.close()
+        self.assertEqual(357, len(parser.items))
```

Comment:
That's a lot of items, this will slow down the tests. Could we make it a smaller sample?

Reply:
Done (2 findings now)